userspace/falco/app/actions/init_inspectors.cpp

```diff
@@ -67,7 +67,13 @@ static void init_syscall_inspector(falco::app::state& s, std::shared_ptr<sinsp>
 			                  "Enabled container runtime socket at '" + p + "' via config file");
 		}
 	}
-	inspector->set_cri_async(!s.config->m_container_engines_disable_cri_async);
+
+	bool disable_cri_async = s.config->m_container_engines_disable_cri_async || s.options.disable_cri_async;
```

Comment:
To make the linter happy:
```suggestion
	bool disable_cri_async = 
		s.config->m_container_engines_disable_cri_async || s.options.disable_cri_async;
```

Reply:
sorry about that, fixed